src/main/java/com/example/auctionapp/customValidator/BidVerifierValidator.java

```diff
@@ -0,0 +1,64 @@
+package com.example.auctionapp.customValidator;
+
+import com.example.auctionapp.Util.TimeUtility;
+import com.example.auctionapp.exception.BadRequestException;
+import com.example.auctionapp.model.Bid;
+import com.example.auctionapp.model.Product;
+import com.example.auctionapp.repository.ProductRepository;
+import org.springframework.beans.BeanWrapperImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import java.util.List;
+
+public class BidVerifierValidator implements ConstraintValidator<BidVerifier, Object> {
```

Comment:
`ConstraintValidator<BidVerifier, Object>` -> `ConstraintValidator<BidVerifier, BidDto>` or whats its name.
Then you can change the method `isValid` to accept BidDTO